app/views/registration/steps/_introduction.html.haml

```diff
@@ -10,27 +10,33 @@
 
   .row{ 'data-equalizer' => true }
     .small-12.medium-12.large-6.columns.pad-top{ 'data-equalizer-watch' => true }
-      %h5= t(".registration_checklist")+":"
-      %ul.check-list
-        %li
-          = t(".registration_time")
-        %li
-          = t(".registration_enterprise_address")
-        %li
-          = t(".registration_contact_details")
-        %li
-          = t(".registration_logo")
-        %li
-          = t(".registration_promo_image")
-        %li
-          = t(".registration_about_us")
+      %input#collapsible-registration-checklist.collapsible-checkbox{:type => "checkbox"}
+      %label.lbl-toggle{:for => "collapsible-registration-checklist"}
+        %h5= t(".registration_checklist")+":"
```

Comment:
Hmm, a collapsed heading "You'll need:" doesn't look right... What do you think of changing the base translation to something similar to "What do I get?" in the second column, such as "What do I need?". To make it simple, it can be the same for both mobile and desktop.

Reply:
I'll do this.

Reply:
https://github.com/openfoodfoundation/openfoodnetwork/pull/3791/commits/8cf7d36d5a161a91f6ff14411ac561dfc6ab8d34